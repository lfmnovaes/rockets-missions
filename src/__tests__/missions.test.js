import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import reducer from '../redux/missions/missions';
import Missions from '../components/MissionList';

describe('Group of Mission tests', () => {
  test('Test if Missions page is rendering correctly', () => {
    const missions = renderer
      .create(
        <Provider store={store}>
          <Missions />
        </Provider>,
      )
      .toJSON();
    expect(missions).toMatchSnapshot();
  });
  test('Check the initialState of the reducer', () => {
    expect(reducer(undefined, {})).toEqual([]);
  });
});
