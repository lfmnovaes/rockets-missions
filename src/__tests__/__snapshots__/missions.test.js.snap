// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group of Mission tests Test if Missions page is rendering correctly 1`] = `
<table
  className="table table-sm table-striped table-bordered table-hover"
>
  <thead>
    <tr>
      <th
        scope="col"
      >
        Name
      </th>
      <th
        scope="col"
      >
        Description
      </th>
      <th
        scope="col"
      >
        Status
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
